plutus-tx/src/Language/PlutusTx/Prelude.hs

```diff
@@ -88,6 +86,208 @@ import           Prelude                    as Prelude hiding (all, any, error,
 -- >>> :set -XNoImplicitPrelude
 -- >>> import Language.PlutusTx.Prelude
 
+-- Copied from the GHC definition
+-- | The 'Eq' class defines equality ('==') and inequality ('/=').
+-- All the basic datatypes exported by the "Prelude" are instances of 'Eq',
+-- and 'Eq' may be derived for any datatype whose constituents are also
+-- instances of 'Eq'.
```

Comment:
But we do not have deriving, right? I mean, you say we have it, but I don't quite understand how and precisely what.

Reply:
Oops, I just copied the Prelude doc, I should remove the bits that aren't true :sweat_smile: